WORKSPACE

```diff
@@ -1,14 +1,24 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+rules_scala_version = "b537bddc58a77318b34165812a0311ef52806318"
+
+
+git_repository(
+    name = "io_bazel_rules_scala",
+    remote = "https://github.com/bazelbuild/rules_scala",
+    commit = "ca5a7acff4ff630f68f58b8e01e8c25dbf908fb7" # HEAD as of 2019-05-13, update this as needed
+)
 
-rules_scala_version="b537bddc58a77318b34165812a0311ef52806318"
 http_archive(
-             name = "io_bazel_rules_scala",
-             url = "https://github.com/bazelbuild/rules_scala/archive/%s.zip"%rules_scala_version,
-             type = "zip",
-             strip_prefix= "rules_scala-%s" % rules_scala_version
-             )
+    name = "com_google_protobuf",
```

Comment:
Why do we need protobuf now?

Reply:
It has nothing to do with us, is a requirement for scala_rules (I don't know why), but it is in their documentation (and certainly, without it, it doesn't compile)